pkg/whitesource/scanMaven.go

```diff
@@ -2,15 +2,18 @@ package whitesource
 
 import (
 	"fmt"
-	"github.com/SAP/jenkins-library/pkg/log"
-	"github.com/SAP/jenkins-library/pkg/maven"
 	"path/filepath"
 	"strings"
+
+	"github.com/SAP/jenkins-library/pkg/log"
+	"github.com/SAP/jenkins-library/pkg/maven"
 )
 
 // ExecuteMavenScan constructs maven parameters from the given configuration, and executes the maven goal
 // "org.whitesource:whitesource-maven-plugin:19.5.1:update".
 func (s *Scan) ExecuteMavenScan(config *ScanOptions, utils Utils) error {
+	s.AgentName = "WhiteSource Maven Plugin"
```

Comment:
I guess we could get rid of the complete file instead.
If I am not mistaken, this is anyway no longer used. We kept it as a safety net during the migration to UA ...